packages/cicada/build.sh

```diff
@@ -0,0 +1,22 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/mitnk/cicada
+TERMUX_PKG_DESCRIPTION="A bash like Unix shell"
+TERMUX_PKG_MAINTAINER="Hugo Wang <w@mitnk.com>"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_VERSION=0.9.11
+TERMUX_PKG_SRCURL=https://github.com/mitnk/cicada/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=a3755aab5c60887587e2e98d0786e7cbf03b9871e931d2a6a177aa093427253d
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+    rm -f Makefile
+}
+
+termux_step_post_make_install() {
+    if [ ! -f "$HOME/.inputrc" ]; then
+        cp -v misc/inputrc $HOME/.inputrc
```

Comment:
This will work for on-device builds but not when cross-compiling the official packages, so put it all inside [a check for `TERMUX_ON_DEVICE_BUILD`](https://github.com/termux/termux-packages/blob/master/packages/swift/build.sh#L136).

Reply:
Updated. Thanks.